### PR TITLE
Added anyHandler

### PR DESCRIPTION
**Pull request by niansa**
Created anyHandler in bot object

### CODE REVIEW

**Comment by niansa** [comment]
Idk why my second commit reverted the first one. Had to revert the revert in a third commit.

**Comment by niansa** [comment]
1 second this stopped working when I hit reconfigure on my IDE :thinking:

**Comment by niansa** [comment]
Ah, forgot to commit a file

**Comment by niansa** [comment]
Fixed

**Comment by niansa** [comment]
Please do not merge yet, I consider changing later.

**Comment by niansa** [comment]
Okay should be good to merge now!

File: discordpp/botStruct.hh [hunk 1]
```diff
@@ -18,6 +18,7 @@ namespace discordpp {
 class BotStruct {
   public:
     std::multimap<std::string, handleEvent> handlers;
+    handleAnyEvent anyHandler = nullptr;
```
Comment (UndarkAido) [inline discordpp/botStruct.hh:21]: Should be an `std::vector` to match the `std::multimap`

**Review by UndarkAido — CHANGES_REQUESTED** [review]

**Review by niansa — COMMENTED** [review]
Yep

**Comment by UndarkAido** [comment]
I'm still conflicted as to whether or not this is a good idea. Could you show me the code where you use this?

**Comment by niansa** [comment]
> I'm still conflicted as to whether or not this is a good idea. Could you show me the code where you use this?

Logging intents, as example.

**Comment by UndarkAido** [comment]
This PR will be implemented as a plugin

**Pull request closed without merging**